stack.py

```diff
@@ -126,3 +126,14 @@ def _merge_list(stack, obj):
         return obj + stack
     else:
         return stack + obj
+
+
+def _parse_top_cfg(content):
```

Comment:
could you rename `_parse_top_cfg` to `_parse_stack_cfg` to be consistent with other naming (`_process_stack_cfg`)